sentry_sdk/integrations/celery.py

```diff
@@ -244,7 +244,7 @@ def _inner(*args, **kwargs):
         if hub.get_integration(CeleryIntegration) is None:
             return f(*args, **kwargs)
 
-        with hub.push_scope() as scope:
+        with hub.configure_scope() as scope:
```

Comment:
the celery test changed because of this, imo where we used `push_scope` before, we need to create a new isolated scope because this is our new forking/isolation logic and the task stuff needs to be isolated.

Reply:
Fixed this. Thanks!